docs/types/access-modifiers.md

```diff
@@ -8,83 +8,54 @@ order: 4
 
 # Access Modifiers
 
-While we don't usually like making things private in a programming language, it
-sometimes the case that it is necessary to indicate that certain fields should
-not be touched (as this might break invariants and such like). To this end, Enso
-provides an explicit mechanism for access modification.
+> [!WARNING]
+> Everybody who ever maintained a large system knows **encapsulation is essential**.
+>
+> While we don't usually like making things private in a programming language, it
+> sometimes the case that it is necessary to indicate that certain fields should
+> not be touched (as this might break invariants and such like). To this end, Enso
+> provides an explicit mechanism for access modification.
+
+Enso targets large user base of _non-programmers_. They are mostly focused on
```

Comment:
Link this section to https://github.com/enso-org/enso/blob/develop/docs/semantics/encapsulation.md ?

Reply:
Done in 4d5f0e69935bc190a0e2dcc87e4df5bdf7ffe009